inc/customizer/assets/scripts/tinymce.js

```diff
@@ -1,11 +1,13 @@
+/* globals tinymce */
```

Comment:
Autoformatted by eslint, had to add a few things to make it work otherwise and pass our coding standards.